app/src/main/res/layout/weather_info_home.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/weather_info_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <FrameLayout
+        android:id="@+id/frame"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="20dp"
+        android:layout_marginBottom="20dp"
+        android:layout_gravity="center"
+        android:background="@drawable/frame_home"
+        >
+        <TextView
+            android:id="@+id/weather_info_text"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            tools:text="esempio"
+            android:textAppearance="@style/home_info_big"
+            android:layout_gravity="center"
+            android:layout_margin="20dp"
+            />
+
+    </FrameLayout>
+
+</FrameLayout>
```

Comment:
All the dimensions need to be moved in dimen.xml file, we do not need to hardcode them into the layout.